internal/server/device/tpm.go

```diff
@@ -227,6 +230,21 @@ func (d *tpm) startVM() (*deviceConfig.RunConfig, error) {
 		return nil, fmt.Errorf("Failed to save swtpm state for device %q: %w", d.name, err)
 	}
 
+	// Wait for the socket to be available.
+	exists := false
+	for i := 0; i < 20; i++ {
+		if util.PathExists(socketPath) {
+			exists = true
+			continue
```

Comment:
Wouldn't `break` be better?

Reply:
doh, yes, that's what I meant to put there ;)

Reply:
Fixed